pom.xml

```diff
@@ -0,0 +1,217 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>20.2</version>
+    </parent>
+
+    <groupId>io.gravitee.ae</groupId>
+    <artifactId>gravitee-alert-engine-connectors</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Gravitee.io - Alert Engine - Connectors</name>
+
+    <modules>
+        <module>gravitee-alert-engine-connector-api</module>
+        <module>gravitee-alert-engine-connector-core</module>
+        <module>gravitee-alert-engine-connector-ws</module>
+    </modules>
+
+    <properties>
+        <gravitee-bom.version>2.4</gravitee-bom.version>
+        <gravitee-node.version>1.24.0</gravitee-node.version>
+        <gravitee-plugin.version>1.23.1</gravitee-plugin.version>
+        <gravitee-alert-api.version>1.9.0</gravitee-alert-api.version>
+        <mockito.version>4.5.1</mockito.version>
+        <junit-jupiter.version>5.8.2</junit-jupiter.version>
+        <assertj.version>3.22.0</assertj.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Import bom to properly inherit all dependencies -->
+            <dependency>
+                <groupId>io.gravitee</groupId>
+                <artifactId>gravitee-bom</artifactId>
+                <version>${gravitee-bom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+
+            <!-- Gravitee.io -->
+            <dependency>
+                <groupId>io.gravitee.cockpit</groupId>
+                <artifactId>gravitee-cockpit-api</artifactId>
+                <version>${gravitee-cockpit-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.gravitee.alert</groupId>
+                <artifactId>gravitee-alert-api</artifactId>
+                <version>${gravitee-alert-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.gravitee.plugin</groupId>
+                <artifactId>gravitee-plugin</artifactId>
+                <version>${gravitee-plugin.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.gravitee.node</groupId>
+                <artifactId>gravitee-node</artifactId>
+                <version>${gravitee-node.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <!-- Test dependencies -->
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-engine</artifactId>
+                <version>${junit-jupiter.version}</version>
```

Comment:
junit-jupiter is imported in bom 2.5, you don't need to specify its version here